docs/source/java/ipc.rst

```diff
@@ -30,7 +30,7 @@ Arrow defines two types of binary formats for serializing record batches:
 
 Writing and Reading Streaming Format
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
-First, let's populate a :class:`VectorSchemaRoot` with a small batch of records
+First, let's populate a :class:`VectorSchemaRoot` with a small batch of records::
```

Comment:
The double-colon is unnecessary in general here since there is an explicit code block anyways.

Reply:
Ok, removed everywhere in the file to be consistent